cylc/flow/cycling/iso8601.py

```diff
@@ -269,8 +269,14 @@ def build_exclusions(self, excl_points):
         for point in excl_points:
             try:
                 # Try making an ISO8601Sequence
-                exclusion = ISO8601Sequence(point, self.exclusion_start_point,
-                                            self.exclusion_end_point)
+                exclusion = ISO8601Sequence(
+                    point,
+                    self.exclusion_start_point,
+                    self.exclusion_end_point,
+                    # disable warnings which are logged when exclusion is a
```

Comment:
Is this comment necessary? Isn't the variable `zero_duration_warning` clear enough?

Reply:
Fair question, it seems to me that "when exclusion is a time point" is useful context. It's necessarily obvious that a zero duration (from the variable name) is equivalent to a time point.